### DISCUSSION TITLE
RuntimeError: decode_jpeg: torchvision not compiled with libjpeg support

### DISCUSSION THREAD

**Discussion by phanducthien82** [mrdbourke/pytorch-deep-learning#686] [Q&A]
Hi, I'm on "04_pytorch_custom_dataset" and I get the error "RuntimeError: decode_jpeg: torchvision not compiled with libjpeg support" when running the command "custom_image_uint8 = torchvision.io.read_image(str(custom_image_path))". Please guide me how to fix this problem. Thank!

**Comment by mrdbourke** [comment]
Hi @phanducthien82 , 

This is interesting, I've never seen this error before.

Which platform are you using with `torchvision`? 

What version of `torchvision` are you using?

This may be because of the latest update to `torchvision` (v0.16.0), see: https://github.com/pytorch/vision/releases/tag/v0.16.0

I'm reading this line: 

<img width="621" alt="screenshot of torchvision 0.16.0 update" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/16750345/b6f02053-ad88-4919-8920-832b6baf5405">

I'll run the notebook on my end and check it out.

-- 

**Update:** I just ran [Notebook 04](https://github.com/mrdbourke/pytorch-deep-learning/blob/main/04_pytorch_custom_datasets.ipynb) in Google Colab and it worked (`torchvision` = 0.16.0+cu118).

Potentially you may want to uninstall and reinstall `torchvision`? 

<img width="802" alt="screenshot of code running in Google Colab" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/16750345/03fb800e-2ad5-4c39-928d-a4976813cf79">

**Reply by phanducthien82** [reply]
Yes, thank you for your reply. I will try it again
